### PR TITLE
Use new series & name override tables

### LINKED ISSUE #1010: Prefer `name_en` over a translated `en_override` name if the monster is in NA
Dependent on https://github.com/TsubakiBotPad/pad-data-pipeline/issues/138

### PR DESCRIPTION
**Pull request by chasehult**
Closes #1010

### CODE REVIEW

File: dbcog/monster_graph.py [hunk 1]
```diff
@@ -48,17 +49,19 @@
...
+  COALESCE(series.series_type, NULL) AS s_series_type,
   exchanges.target_monster_id AS evo_gem_id,
   drops.drop_id
 FROM
   monsters{0}
   LEFT OUTER JOIN leader_skills{0} ON monsters{0}.leader_skill_id = leader_skills{0}.leader_skill_id
   LEFT OUTER JOIN active_skills{0} ON monsters{0}.active_skill_id = active_skills{0}.active_skill_id
-  LEFT OUTER JOIN series ON monsters{0}.series_id = series.series_id
+  LEFT OUTER JOIN monster_series ON monsters{0}.monster_id = monster_series.monster_id AND priority = 1
+  LEFT OUTER JOIN series ON monster_series.series_id = series.series_id
```
Comment (RheingoldRiver) [inline dbcog/monster_graph.py:64]: should monster_name_overrides be in this table list?

File: dbcog/monster_graph.py [hunk 2]
```diff
@@ -48,17 +49,19 @@
...
   active_skills{0}.turn_min,
-  series.name_ja AS s_name_ja,
-  series.name_en AS s_name_en,
-  series.name_ko AS s_name_ko,
-  series.series_type AS s_series_type,
+  COALESCE(series.series_id, 0) AS s_series_id,
+  COALESCE(series.name_ja, 'Unsorted') AS s_name_ja,
+  COALESCE(series.name_en, 'Unsorted') AS s_name_en,
+  COALESCE(series.name_ko, 'Unsorted') AS s_name_ko,
+  COALESCE(series.series_type, NULL) AS s_series_type,
```
Comment (RheingoldRiver) [inline dbcog/monster_graph.py:56]: coalesce with NULL doesn't do anything

File: dbcog/models/monster_model.py [hunk 3]
```diff
@@ -42,7 +42,11 @@ def __init__(self, **m):
         else:
             # Remove annoying stuff from NA names, like Jörmungandr
             self.name_en = tsutils.rmdiacritics(self.name_en)
-        self.name_en_override = m['name_en_override']
+
+        self.name_en_override = None
+        if not m['name_is_translation'] or self.name_en == self.name_ja:
```
Comment (RheingoldRiver) [inline dbcog/models/monster_model.py:47]: im super confused by this if statement, see discord comments

File: dbcog/monster_graph.py (hunk 1 above)
Comment (chasehult) [reply dbcog/monster_graph.py:64]: It's the last join

File: dbcog/monster_graph.py (hunk 2 above)
Comment (chasehult) [reply dbcog/monster_graph.py:56]: Series 0 has None as it's series_type, so it's just there for consistency.  I can remove if you'd like

**Pull request merged**